src/badge/snippets/badge-on-icon-text-demo.html

```diff
@@ -1,12 +1 @@
-<style>
```

Comment:
I see where you're coming from in removing duplicate styles, but this assumes that we'll always have the same structure, in that we'll always load all demos into a single page. This may or may not be the case when we revamp component tests, and if we decide to go single-page-per-snippet, we'd be missing the styling here. We'd also be making one snippet dependent on another snippet's styling, which could change if we decide to alter one of the demos.

Should we define the style per snippet rather than per component in this case? This way we'd avoid duplicate styles while still making sure each file has its own styles.